lib/pages/laws.dart

```diff
@@ -5,47 +5,169 @@ import 'package:heroes_manual/utility/colors.dart';
 
 class Laws extends StatelessWidget {
   static const String route = '/laws';
+  static const List<String> GA_laws = [
+    'Family Violence Act:\N'
+      + 'The law protects against physical, sexual, and emotional abuse among '
+        + 'family members.\N'
```

Comment:
the escape characters have to be lowercase "\n". Please proof your code before you push. There are capital Ns everywhere in the text on the app